ui/app/templates/components/toolbar-secret-link.hbs

```diff
@@ -1,17 +1,10 @@
 <SecretLink
-  @class="toolbar-link"
-  @mode={{this.mode}}
-  @secret={{this.secret}}
-  @replace={{this.replace}}
-  @queryParams={{this.queryParams}}
-  @data-test-edit-link={{this.data-test-edit-link}}
-  @data-test-sign-link={{this.data-test-sign-link}}
-  @data-test-transit-link={{this.data-test-transit-link}}
-  @data-test-secret-create={{this.data-test-secret-create}}
-  @data-test-credentials-link={{this.data-test-credentials-link}}
-  @data-test-backend-credentials={{this.data-test-backend-credentials}}
-  @data-test-transit-action-link={{this.data-test-transit-action-link}}
-  @data-test-transit-key-actions-link={{this.data-test-transit-key-actions-link}}
+  @mode={{@mode}}
+  @secret={{@secret}}
+  @replace={{@replace}}
+  @queryParams={{@queryParams}}
+  class="toolbar-link"
+  ...attributes
```

Comment:
Since we're passing along attributes named the same, would those all be captured in `...attributes` so we can remove all the `@attrs`? If we do this, I'd love to see those attributes called out in the JSDocs of the class component

Reply:
Sorry I'm not sure I fully understand. What do you mean by `@attrs`? `...attributes` will only spread html attributes. I thought I saw an Ember RFC for being able to spread component args as well which would be great but not sure when that will land. As far as attributes named the same, the values are concatenated. For example:
```hbs
<ToolbarSecretLink
 class="passed-in"
/>
{{!-- results in --}}
<a href="/ui/vault/secrets/secret/create" id="ember509" class="ember-view toolbar-link passed-in" data-test-secret-create="">
  Create secret
</a>
```

Reply:
[Found the RFC I mentioned!](https://github.com/emberjs/rfcs/pull/593)

Reply:
Oh nice! I think i was thinking about React-land, where non-html attributes are passed. Carry on!